README.md

```diff
@@ -5,7 +5,7 @@ This is the documentation of Intrepid best practices and style regarding the Swi
 
 ##Making / Requesting Changes
 
-Feedback and changes are encouraged!  The current maintainers of this style guide are @brightredchili and @loganwright.  If you have an issue with any of the existing rules and would like to see something changed, please see the [contribution guidelines](CONTRIBUTING.md),
+Feedback and change are encouraged!  The current maintainers of this style guide are @brightredchili.  If you have an issue with any of the existing rules and would like to see something changed, please see the [contribution guidelines](CONTRIBUTING.md),
```

Comment:
I'm clearly not the only one mantaining it - while you're making this change, can you please change to 'developers of Intrepid'? ❤️